src/Tables/molecules/TypeTableHeader.tsx

```diff
@@ -1,12 +1,12 @@
-import React, { useState, useCallback, Fragment } from 'react';
+import React, { useState, useCallback, Fragment} from 'react';
 import styled, { css } from 'styled-components';
 import { TypeCellAlignment, ITableColumnConfig } from '..';
 import Checkbox from '../../Form/atoms/Checkbox';
 import TableHeaderTitle from '../atoms/TableHeaderTitle';
 
-const HeaderRow = styled.div<{ HEADER_HEIGHT: string }>`
+const HeaderRow = styled.div`
   display: table-row;
-  height: ${({ HEADER_HEIGHT }) => HEADER_HEIGHT};
+  height: 50px;
```

Comment:
height behaves like min-height on table elements, so this one can stay like 50px;